data/huion-hs610.tablet

```diff
@@ -0,0 +1,50 @@
+# HUION
+# HS610
+#
+# Button Map:
+#      *-----------------------------------*
+#      |                                   |
+#  A|B |                                   |
+#  C|D |                                   |
+#  E|F |                                   |
+#      |                                   |
+#   M  |              TABLET               |
+#      |                                   |
+#  G|H |                                   |
+#  I|J |                                   |
+#  K|L |                                   |
+#      |                                   |
+#      *-----------------------------------*
+#
+# Touch Ring Map:
+# (A=1st ring, B=2nd ring, ...)
+#
+#    *-----------------------*
+#    |                       |
+#  M |        TABLET         |
+#    |                       |
+#    *-----------------------*
+#
+
+[Device]
+Name=HUION Huion Tablet
+ModelName=HS610
+DeviceMatch=usb:256c:006d
```

Comment:
Mine says
```
[Device]
Name=Huion H950P
ModelName=
DeviceMatch=usb:256c:006d
```

Reply:
That's the one we added in the data file, right? I'll need the kernel device name because that is what comes from the device itself. `sudo libinput record` will list all kernel evdvev nodes, or you can get it out of `/proc/bus/input/devices`/ Apologies if this wasn't clear.

Reply:
```
I: Bus=0003 Vendor=256c Product=006d Version=0111
N: Name="HID 256c:006d Pen"
P: Phys=usb-0000:00:14.0-2/input0
S: Sysfs=/devices/pci0000:00/0000:00:14.0/usb2/2-2/2-2:1.0/0003:256C:006D.0004/input/input28
U: Uniq=
H: Handlers=mouse3 event20 
B: PROP=2
B: EV=1b
B: KEY=1c01 0 0 0 0 0
B: ABS=1000003
B: MSC=10

I: Bus=0003 Vendor=256c Product=006d Version=0111
N: Name="HID 256c:006d Pad"
P: Phys=usb-0000:00:14.0-2/input0
S: Sysfs=/devices/pci0000:00/0000:00:14.0/usb2/2-2/2-2:1.0/0003:256C:006D.0004/input/input29
U: Uniq=
H: Handlers=event21 js0 
B: PROP=0
B: EV=1b
B: KEY=800 30000000003ff 0 0 0 0
B: ABS=3
B: MSC=10
```
From /proc/bus/input/devices, Pen and Pad.

Reply:
@spbnick - do you happen to know whether we can make the Huion H950P set the name to something more useful?

Reply:
Unfortunately not. I wouldn't bet on the name staying the same across releases of the same model either. In theory the kernel driver could expose the binary tablet parameters as, say, a hexdump string via the evdev interface somehow. That is not guaranteed to stay the same either, though. E.g. there's been tablet firmware updates adding tilt sensing which wrecks havoc to *that*, although that's a rare occurrence.

Reply:
Here's my /proc/bus/input/devices for the Pen and Pad:
`I: Bus=0003 Vendor=256c Product=006d Version=0111
N: Name="HUION Huion Tablet Pen"
P: Phys=usb-0000:02:00.0-5/input0
S: Sysfs=/devices/pci0000:00/0000:00:01.2/0000:02:00.0/usb1/1-5/1-5:1.0/0003:256C:006D.0002/input/input10
U: Uniq=
H: Handlers=mouse2 event8 
B: PROP=2
B: EV=1b
B: KEY=1c01 0 0 0 0 0
B: ABS=1000003
B: MSC=10

I: Bus=0003 Vendor=256c Product=006d Version=0111
N: Name="HUION Huion Tablet Pad"
P: Phys=usb-0000:02:00.0-5/input0
S: Sysfs=/devices/pci0000:00/0000:00:01.2/0000:02:00.0/usb1/1-5/1-5:1.0/0003:256C:006D.0002/input/input11
U: Uniq=
H: Handlers=event9 js0 
B: PROP=0
B: EV=1b
B: KEY=800 30000000003ff 0 0 0 0
B: ABS=3
B: MSC=10
`

Reply:
>  I wouldn't bet on the name staying the same across releases of the same model either.

sigh, that'll make supporting the huion tablets in libwacom tricky. thanks though.

Ok, for now let's add the 610 with the full name and leave the 950 as-is with it's HID name. And then fix it when we have to in the future :(

@filoumoteur: please add the name to the `DeviceMatch`. see the `data/gaomon-s620.tablet` file for an example.